.travis.yml

```diff
@@ -1,8 +1,22 @@
 language: go
 
+os:
+  - linux
+  - osx
+
 go:
   - 1.5.1
 
+env:
+  - TARGETS="check testsuite"
+#  - TARGETS="crosscompile"
```

Comment:
We never had that capability before for Packetbeat. `gopacket` uses cgo so we would have to have the gcc toolchain available for each os/arch.